### PR TITLE
Add virtual Arduino docs

### PR DESCRIPTION
**Pull request by tpoomlmly**
Covered simulator differences, analogue + digital IO.

Need to make changes to `arduino.md:31` to match @Adimote 's changes once the virtual robot docs are complete

### CODE REVIEW

File: content/api/arduino.md [hunk 1]
```diff
@@ -21,7 +21,64 @@ from microswitches to LEDs. GPIO is only available on pins 2 to 13 and
...
 
-## Pin mode
+## Simulator
+
+In the simulator, the Arduino's pins are pre-populated and pre-configured.
+The first few digital pins are occupied by digital inputs, the next few by
+digital outputs, and the analogue pins are attached to distance sensors.
+
+To find out how many inputs and outputs each type of robot has, check the
+[robot docs](TODO_change_when_this_page_is_added).
```
Comment (Adimote) [inline content/api/arduino.md:31]: added in #174  :)

File: content/api/arduino.md [hunk 2]
```diff
@@ -113,7 +170,7 @@ The values are the voltages read on the pins, between 0 and 5.
 Pins `A4` and `A5` are reserved and cannot be used.
 {{% /notice %}}
 
-## Ultrasound Sensors
+## Ultrasound Sensors (Unavailable in Simulator)
```
Comment (Adimote) [inline content/api/arduino.md:173]: I think we're calling the distance sensors ultrasound sensors in the simulator (they have an ultrasound sensor model, and behave like an ultrasound sensor, except they don't have a minimum distance).

**Review by Adimote — APPROVED** [review]

File: content/api/arduino.md (hunk 2 above)
Comment (RealOrangeOne) [reply content/api/arduino.md:173]: Correct. The API is also slightly different.

File: .spelling [hunk 3]
```diff
@@ -61,4 +61,7 @@ vcc
 VSCode
 W3Schools
 Webots
+www.python.org
```
Comment (RealOrangeOne) [inline .spelling:64]: These items don't appear to be used in this PR?

File: content/api/arduino.md [hunk 4]
```diff
@@ -21,7 +21,64 @@ from microswitches to LEDs. GPIO is only available on pins 2 to 13 and
...
+Arduino's analogue input pins starting from pin `A0`. You can read their
+values like this:
+
+```python
+distance = r.arduino.pins[AnaloguePin.A0].analogue_value
+```
+
+The value read is returned as a float.
+
+### Distance Sensor
```
Comment (RealOrangeOne) [inline content/api/arduino.md:72]: I think we need to document here how the physical robot API works for this, as it's slightly different.

File: .spelling (hunk 3 above)
Comment (tpoomlmly) [reply .spelling:64]: they're not, but my local copy of the spellchecker was flagging them up

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**